docs/core/tools/dotnet.md

```diff
@@ -19,9 +19,13 @@ ms.technology: dotnet-cli
 ## Synopsis
 
 # [.NET Core 2.x](#tab/netcore2x)
+
 `dotnet [command] [arguments] [--additional-deps] [--additionalprobingpath] [-d|--diagnostics] [--fx-version] [-h|--help] [--info] [--roll-forward-on-no-candidate-fx] [-v|--verbose] [--version]`
+
 # [.NET Core 1.x](#tab/netcore1x)
+
 `dotnet [command] [arguments] [--additionalprobingpath] [-d|--diagnostics] [--fx-version] [-h|--help] [--info] [-v|--verbose] [--version]`
+
```

Comment:
That's how it was before, no @JRAlexander? I'll change to the code block. It works on the other topics like that.